game.rb

```diff
@@ -0,0 +1,36 @@
+require 'json'
+require_relative 'item'
+
+class Game < Item
+
+  attr_accessor :multiplayer, :last_played_at, :publish_date
+  def initialize(multiplayer, last_played_at, publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = Date.parse(last_played_at)
+    @publish_date = publish_date
+  end
```

Comment:
Kindly include all the properties in the constructor parameters (id, archived... etc) using the constructor method from the parent class (`super()`)